server/src/main/java/org/bonitasoft/console/common/server/servlet/FileUploadServlet.java

```diff
@@ -180,6 +192,14 @@ public void doPost(final HttpServletRequest request, final HttpServletResponse r
         }
     }
 
+    /**
+     * @param fileItemFactory
+     * @return
+     */
+    protected ServletFileUpload createServletFileUpload(final FileItemFactory fileItemFactory) {
```

Comment:
Uneeded comment.
I would prefer having something saying that it is protected for testing purpose